mmv1/templates/terraform/resource.erb

```diff
@@ -1060,6 +1059,7 @@ func resource<%= object.resource_name -%>Delete(d *schema.ResourceData, meta int
       billingProject = bp
     }
 
+<%= lines(compile(pwd + '/' + object.custom_code.pre_delete)) if object.custom_code.pre_delete -%>
```

Comment:
This will be removed after https://github.com/GoogleCloudPlatform/magic-modules/pull/10143 is merged

Reply:
That's done!